cachito/errors.py

```diff
@@ -2,11 +2,11 @@
 from enum import Enum
 
 
-class RequestErrorOrigin(Enum):
+class RequestErrorOrigin(str, Enum):
     """An Enum that represents the request error origin."""
 
-    client = 1
-    server = 2
+    client = "client"
+    server = "server"
```

Comment:
Can we make this change in first commit `Define all error types` ?

Reply:
Yep, I reorganized edits so that they belong to their respective commits.

Reply:
Thank you sir :)